src/main/scala/xiangshan/frontend/FrontendBundle.scala

```diff
@@ -125,9 +132,7 @@ class FetchToIBuffer(implicit p: Parameters) extends XSBundle {
   val foldpc    = Vec(PredictWidth, UInt(MemPredPCWidth.W))
   val ftqPtr       = new FtqPtr
   val ftqOffset    = Vec(PredictWidth, ValidUndirectioned(UInt(log2Ceil(PredictWidth).W)))
-  val ipf          = Vec(PredictWidth, Bool())
-  val igpf          = Vec(PredictWidth, Bool())
-  val acf          = Vec(PredictWidth, Bool())
+  val exceptionType = Vec(PredictWidth, UInt(2.W))
```

Comment:
2.W seems a bit magical to me. Can this be defined as ExceptionType.width or something similar?

Reply:
Fair enough.